index.js

```diff
@@ -6,7 +6,8 @@ const { join } = require('path'),
   { readdirSync } = require('fs');
 
 module.exports = {
-  plugins: ['jsdoc', 'lodash', 'mocha', 'node', 'security', 'sonarjs'],
+  plugins: ['jsdoc', 'lodash', 'mocha', 'node', 'security', 'sonarjs', '@typescript-eslint'],
+  parser: '@typescript-eslint/parser',
```

Comment:
1. Will this have any side-effects on JS code like transpiling plain JS with TS does?
2. Do we control which target ECMA standard version is being used by this parser?

Reply:
1. Let me move it to overrides to prevent it from parsing JS files.
2. It [respects](https://typescript-eslint.io/architecture/parser#configuration) the parserOptions on line 17.